doctr/models/modules/transformer/pytorch.py

```diff
@@ -108,6 +108,7 @@ def __init__(
         num_layers: int,
         num_heads: int,
         d_model: int,
+        dff: int,
```

Comment:
Let's add a docstring here or at least the Transformer paper reference for the notations. `dff` is not explicit enough :sweat_smile: Other args are fine

Reply:
Used only for internal stuff / building models .. i have added a comment on it this should be enough :)

Reply:
Thanks @felixdittrich92 !